src/templater.sh

```diff
@@ -48,6 +48,7 @@ function templater() {
 
   # cat "${templateDst}"
 
+  # shellcheck disable=SC2016
   if grep '${' "${templateDst}" && [ "${ALLOW_REMAINDERS}" = "0" ]; then
```

Comment:
fun. that error message didn't look like a simple restart would change anything.
also i have no restart option. i'm guessing because i'm not the owner of anything there.